rules/validator.go

```diff
@@ -164,11 +169,17 @@ func (v *validator) validateFilters(f map[string]string) error {
 }
 
 func (v *validator) validateRollupTags(ruleName string, tags []string) error {
-	requiredTags := v.opts.RequiredRollupTags()
-	if len(requiredTags) == 0 {
-		return nil
+	// Validate that all tags have valid characters
+	invalidChars := v.opts.RollupTagInvalidChars()
+	for _, tag := range tags {
+		fmt.Println("in here")
+		if err := validateChars(tag, invalidChars); err != nil {
+			return fmt.Errorf("invalid rollup tag %s for rollup rule %s : %s", tag, ruleName, err.Error())
```

Comment:
Perhaps make the error message consistent w/ others, e.g., something along the lines of `fmt.Errorf("rollup rule %s has invalid rollup tag %s: %v", ruleName, tag, err)`?